src/components/visual-tooltip/VisualTooltipContent.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+import './VisualTooltipContent.scss';
+
+export type VisualTooltipContentProps = {
+    /** A string to be used in the tooltip's paragraph content */
+    content: string;
+    /** The string source of the image used in the visual tooltip */
+    imageSrc: string;
+    /** A string to be used in the tooltip's title heading */
+    title: string;
+};
+
+const VisualTooltipContent = ({ content, imageSrc, title }: VisualTooltipContentProps) => (
+    <div className="bdl-VisualTooltipContent">
+        <div className="bdl-VisualTooltipContent-imageWrapper">
+            <img src={imageSrc} alt={title} className="bdl-VisualTooltipContent-image" />
```

Comment:
Should the `alt` for the image always necessarily be the same as `title`?

Reply:
Maybe instead, could you pass the image as a `React.Node` prop? Instead of the `bdl-VisualTooltipContent-image` styling for `max-width`, could you style `bdl-VisualTooltipContent-imageWrapper` as a flexbox container that stretches content to full width?

Reply:
Looking into the flexbox recommendation now. I suppose I can just change it to a `React.Node` and then let the parent pass in whatever they want.